docs/Learning-Environment-Design-Agents.md

```diff
@@ -560,6 +562,36 @@ between -1 and 1.
  of an entity to the `BufferSensor`.
  - Normalize the entities observations before feeding them into the `BufferSensor`.
 
+### Goal Signals
+
+It is possible for agents to collect observations that will be treated as "goal."
```

Comment:
```suggestion
It is possible for agents to collect observations that will be treated as a "goal."
```

Reply:
I think within this paragraph we should also call these goal signals, and not goals.